project/Settings.scala

```diff
@@ -323,6 +324,12 @@ object Settings {
       Seq("edu.gemini" %%% "refined-algebra" % refinedAlgebra)
     )
 
+    val ScalaCollectionContrib = Def.setting(
+      deps(
+        "org.scala-lang.modules" %%% "scala-collection-contrib"
```

Comment:
What are we using this for? I thought this mostly needed for cross built libraries

Reply:
For `list.intersperse(...)`.